place_and_route/open_road.bzl

```diff
@@ -245,14 +245,22 @@ def openroad_command(ctx, commands, input_db = None, step_name = None, inputs =
         ),
     )
 
+    # Don't overwhelm remote executors (which may be running multiple actions)
+    # with too many threads. Just use one unless the job is required to be run
+    # locally, in which case we expect an action will often have the entire
+    # system to itself.
+    num_threads = "1"
+    if "no-remote-exec" in execution_requirements:
```

Comment:
Will this trigger if you run with --strategy=OpenROAD=local?

Reply:
It looks like setting the strategy still keeps num_threads=1 :/

Might be better to have num_threads be a parameter to override. WDYT?

Reply:
Could we check the opposite? i.e. is this running remotely?